com.transmissionbt.Transmission.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id": "com.transmissionbt.Transmission",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "transmission-gtk",
+    "rename-desktop-file": "transmission-gtk.desktop",
+    "rename-icon": "transmission",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
```

Comment:
Have you asked upstream about why Qt frontend claims different name than GTK? My guess is that they should unify this. It is also preferred not to maintain patches downstream if possible, but yes, we could modify it ourselves, whatever works for you.